recipes/winregistry/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "winregistry" %}
```

Comment:
Can you make a separate PR for this package? Otherwise, we will not be able to get to a green CI on any platform.

Reply:
sure thing

Reply:
heres the PR #28401